supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -125,14 +125,30 @@ For example, instead of "Red Hat recommends using X package because", write "Use
 [discrete]
 [[red-hat-container-catalog]]
 ==== Red Hat Container Catalog (noun)
-*Description*: "Red Hat Container Catalog" is Red Hat's hosted registry for Enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers]. Red Hat Container Catalog is a trusted source for secure, certified, and up-to-date container images. Write this name in full the first time that you use it in a document.
+*Description*: "Red Hat Container Catalog" was the Red Hat hosted registry for enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers].
```

Comment:
Should it be Red Hat-hosted registry?

Reply:
@leswilliams44 Updated, if you can re-review please